internal/service/alertconfiguration/data_source_alert_configuration.go

```diff
@@ -1,22 +1,24 @@
-package mongodbatlas
+package alertconfiguration
```

Comment:
since alertconfiguration is in the package name, can we remove alertconfiguration from the name of the files in the package

Reply:
would keep in mind this is useful for when you search files across the whole repo

Reply:
do you mean to say just data_source.go?

Reply:
and plural, data_sources?

Reply:
not sure about this, i think it's better to keep it in the name, imagine in VSCode having several files opened all called data_source.go

Reply:
> would keep in mind this is useful for when you search files across the whole repo

when you are searching a file in the IDE you also see the package name (at least this is what I see in vscode/intellJ)


> not sure about this, i think it's better to keep it in the name, imagine in VSCode having several files opened all called data_source.go

Makes sense. I just want the team to consider this. Happy to resolve this thread. Thanks for the discussion